docs/self-managed/platform-architecture/overview.md

```diff
@@ -0,0 +1,36 @@
+---
+id: overview
+title: "Camunda Platform 8 Self-Managed architecture"
+sidebar_label: "Overview"
+---
+
+Camunda Platform 8 Self-Managed is highly customizable and can be deployed in different setups.
+[**Helm/Kubernetes**](../platform-deployment/helm-kubernetes/overview.md) is the recommended method to deploy Camunda Platform 8, especially in production. Our Helm chart provides many capabilities to customize deployment according to your needs. This page provides a high-level overview of the architecture and deployment options.
+
+## Architecture
+
+Camunda Platform 8 Self-Managed has multiple web applications and gRPC services which could be accessed externally using Ingress. The following diagram is the architecture with two Ingress objects, an Ingress with HTTP(S) protocol for all web applications using a single domain, and another Ingress with gRPC protocol for the Zeebe workflow engine:
+
+![Camunda Platform 8 Self-Managed Architecture Diagram - Combined Ingress](./assets/camunda-platform-8-self-managed-architecture-diagram-combined-ingress.png)
+
+In this architecture, Camunda Platform 8 Self-Managed can be accessed as follows:
+
+- Web applications: https://camunda.example.com/[identity|operate|optimize|tasklist]
```

Comment:
Non-blocking, but I wonder if there is a way we can turn this text into a link like the others below?

Reply:
The points here are not meant to be links but should be shown as they are (as an example).
Maybe we make them like this:

- Web applications: `https://camunda.example.com/[identity|operate|optimize|tasklist]`

Reply:
OH -- I see what you're saying. I'll revert the others 👍